pkg/kv/kv.go

```diff
@@ -32,6 +35,141 @@ import (
 
 var extraHandleColumnInfo = model.NewExtraHandleColInfo()
 
+// Iter abstract iterator method for Ingester.
+type Iter interface {
+	// Seek seek to specify position.
+	// if key not found, seeks next key position in iter.
+	Seek(key []byte) bool
+	// Error return current error on this iter.
+	Error() error
+	// First return the first key in this iter.
+	First() []byte
```

Comment:
```suggestion
	First() bool
```
Return a bool to indicate if the key is valid